runtime/wasm/src/module/mod.rs

```diff
@@ -472,6 +472,12 @@ impl<C: Blockchain> WasmInstance<C> {
         //
         // For reference, search this codebase for: ff652476-e6ad-40e4-85b8-e815d6c6e5e2
         link!("ipfs.cat", ipfs_cat, "host_export_ipfs_cat", hash_ptr);
+        link!(
+            "ipfs.getBlock",
+            ipfs_get_block,
+            "host_export_ipfs_get_block",
+            hash_ptr
+        );
```

Comment:
You can gate the function linking by `allow_non_deterministic_ipfs`.

Reply:
Done